index.js

```diff
@@ -80,11 +80,14 @@ function processAll() {
     if (e) {
       console.error(e);
     }
+    var total = _.reduce(results, function(sum, num) {return _.zipWith(sum, num, function (v1, v2) { return v1 + v2 })}, [0, 0]);
```

Comment:
I console.logged it, and it showed what I expected it to show.

results is an array of arrays like
[ [x1, y1],
  [x2, y2],
  [x3, y3] ]
Where each inner [x, y] represents one table, and the two values are for read and write throttle matrix. The _.reduce function now takes the inner [x, y] and adds them one by one to the start [0, 0], the adding of an inner [x, y] to [totalX, totalY] is done by the zipWith function as you came up with.